README.md

```diff
@@ -101,14 +101,13 @@ Developers on **GitHub** can call the **GitHub Action** to lint their code base
 | **YAML**                         | [YamlLint](https://github.com/adrienverge/yamllint)                                                                                                                           |
 
 ### 23andMe Custom Linters
-| _Language_                       | _Linter_                                                                                                                                                                      |
+| _Language_                       | _Linter_                                            |
 | -------------------------------- | ----------------------------------------------------|
-| _(All files)_                    | [gitleaks](https://github.com/zricethezav/gitleaks) |
+| _(All files)_                    | [gitleaks](https://github.com/zricethezav/gitleaks) / [deps-checker]   (https://github.com/23andMe/super-linter/blob/master/ttam-linters/deps-checker.sh)       |     
```

Comment:
Should this bee `All files` or dependency files?

Reply:
Yes actually  it should be categorized  as `deps files` initially we thought it will going to look for all the files and will filter out only dependency files. But now I feel like it should be under dependency files.